Dis 836 admin control over ce leaderboard

AlexanderBlanchardAC opened the pull request:
https://aspen-discovery.atlassian.net/browse/DIS-836

AlexanderBlanchardAC committed:
DIS-836: feat: Add control to library system

Add control over whether or not to include campaign leaderboard to
Library Systems

AlexanderBlanchardAC committed:
DIS-836: feat: Add campaign leaderboard control to db

AlexanderBlanchardAC committed:
DIS-836: feat: Display leaderbaord link conditionally

Add a condition to the display of the campaign leaderboard link

AlexanderBlanchardAC committed:
DIS-836: Add No access statement

AlexanderBlanchardAC committed:
DIS-836: chore: Update change log

Jacobomara901 commented:
Test plan follows as expected mate, though if it's not possible to do this without having webbuilder enabled I think that should be added to the enable checkbox as a dependancy, otherwise people won't know and it will cause confusion. Obviously not having the deb would be preferable but I'm assuming that's not possible?

Jacobomara901 commented:
Code looks good as always

AlexanderBlanchardAC committed:
DIS-836: refactor: Add note to setting in library

Add a note to the settings description to warn users that the Web
Builder must be enabled in order for the leaderboard link to show.

AlexanderBlanchardAC commented:
Yeah, Web Builder dependency has to remain. I have added a note to the description of the setting to warn users that it needs to be enabled. :)
